tests/publiccloud/download_repos.pm

```diff
@@ -25,6 +25,13 @@ sub run {
     assert_script_run("mkdir ~/repos");
     assert_script_run("cd ~/repos");
 
+    # Shortcut for not doing updates (useful for speedup of verification runs)
+    if (get_var('PUBLIC_CLOUD_SKIP_DOWNLOAD') == 1) {
+        record_info("Skipping download of repositories", "Skipping update repos because PUBLIC_CLOUD_SKIP_DOWNLOAD=1");
+        assert_script_run('touch /tmp/repos.list.txt');
```

Comment:
I checked the code this file which you are touching is not used anywhere except this module which we actually skipping. Do you have good reason to touch it ?

Reply:
To speed up the verification runs. We could move this to another PR though ....